tfc_docs.cmake

```diff
@@ -30,3 +30,156 @@ function(tfc_add_example EXAMPLE_TARGET cpp_file)
     ${EXAMPLE_TARGET}
   )
 endfunction()
+
+function(tfc_add_docs FILE_PATH DOCS_DIR)
+  file(GLOB FILES_TO_COPY "${FILE_PATH}/*")
+  file(COPY ${FILES_TO_COPY} DESTINATION ${DOCS_DIR})
+endfunction()
+
+function(set_docs SPHINX_SOURCE ENABLE_DOXYGEN)
```

Comment:
I don't find this name explaining its intention, would `tfc_docs_init` be better?

Reply:
This is irrelevant if suggestion below is implemented

Reply:
I agree, tfc_docs_init is a better name